atd-vze/src/views/Locations/Locations.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { withApollo } from "react-apollo";
+
+import GridTable from "../../Components/GridTable";
+
+import gqlAbstract from "../../queries/gqlAbstract";
+
+// Our initial query configuration
+let queryConf = {
+  table: "atd_txdot_locations",
+  single_item: "locations",
+  columns: {
+    unique_id: {
+      primary_key: true,
+      searchable: true,
+      sortable: true,
+      label_search: "Search by location id",
+      label_table: "Location ID",
+      type: "Int",
+    },
+    description: {
+      searchable: true,
+      sortable: false,
+      label_search: "Search by intersecting street name",
+      label_table: "Intersection",
+      type: "String",
+    },
+  },
+  order_by: {
+    unique_id: "desc", // Unique ID desc by default
+  },
+  where: {},
+  limit: 25,
+  offset: 0,
+};
+
+let locationsQuery = new gqlAbstract(queryConf);
```

Comment:
🔥

Reply:
Should we port this abstraction over to Crashes? What other places could it be used? Will you make a new issue for the backlog where this could be reused?